src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp

```diff
@@ -34,9 +34,12 @@
 
 	AddMessageFilter < BsmMessage > (this, &ODEForwardPlugin::HandleRealTimePublish);
  	AddMessageFilter < SpatMessage > (this, &ODEForwardPlugin::HandleSPaTPublish);
+	AddMessageFilter < MapDataMessage > (this, &ODEForwardPlugin::HandleMapPublish);
```

Comment:
In the future we may want to also include a configuration parameter that sets which messages we forward. Seems weird to be able to configure the communication protocol but not messages but probably not a blocker for the planned demo